schemas/questionnaire_v1.json

```diff
@@ -71,12 +71,33 @@
               "optional_string",
               "uuid"
             ]
+          },
+          "length": {
+            "type": "object",
+            "properties": {
+              "min": {
+                "type": "number",
+                "description": "The minimum length of a string field"
+              },
+              "max": {
+                "type": "number",
+                "description": "The maximum length of a string field"
+              },
+              "equal": {
+                "type": "number",
+                "description": "The length of a string field must be equal to this"
```

Comment:
I don't see the need to nest properties under `length`. If we were to extend the validation to numbers it's not clear what the nesting property would be (`size` possibly, but that looks a bit weird). Also, using `min_length` and `max_length` would more closely relate to json schema rather than marshmallow?